Sources/Turf/GeoJSON.swift

```diff
@@ -60,3 +65,23 @@ extension GeoJSONObject: Codable {
         }
     }
 }
+
+/**
+ A type that can be represented as a `GeoJSONObject` instance.
+ */
+public protocol GeoJSONObjectConvertible {
```

Comment:
Below, you made `Geometry` conform to the corresponding protocol. Should we do the same here with `GeoJSONObject`?

Reply:
I finally realized what “idempotent” means. 😄